### PR TITLE
Moderation: Properly log /hidetext's message

### PR DESCRIPTION
**Pull request by mia-pi-git**

### CODE REVIEW

**Comment by Zarel** [comment]
```
// if the user hiding their own text, it would clear the "cleared" message,
// so we can't attribute it in that case
```

How did you deal with this issue?

**Comment by mia-pi-git** [comment]
> ```
> // if the user hiding their own text, it would clear the "cleared" message,
> // so we can't attribute it in that case
> ```
> 
> How did you deal with this issue?

It adds the mod action after room.hideText is run.

**Comment by Zarel** [comment]
> It adds the mod action after `room.hideText` is run.

Wouldn't that put the "show hidden messages" button in the wrong place?

**Comment by mia-pi-git** [comment]
> > It adds the mod action after `room.hideText` is run.
> 
> Wouldn't that put the "show hidden messages" button in the wrong place?

Yes. Resolved by just using `this.roomlog`. Edited the comment as requested.

File: server/chat-commands/moderation.ts [hunk 1]
```diff
@@ -1660,13 +1660,13 @@ export const commands: ChatCommands = {
 
 		// if the user hiding their own text, it would clear the "cleared" message,
 		// so we can't attribute it in that case
+		// and sending the message after the hideText call puts the "show lines" button in the wrong place
```
Comment (Zarel) [inline server/chat-commands/moderation.ts:1663]:
```suggestion
		// and sending the message after `|unlink|` puts the "show lines" button in the wrong place
```

**Comment by Zarel** [comment]
ty!

**Pull request merged**